libfuzzer.cc

```diff
@@ -181,6 +179,12 @@ void Fuzz() {
   args.push_back(nullptr);
   char** args_ptr = &args[0];
   int args_size = args_global.size();
+  
+  if (num_counters) {
+    counters = new unsigned char[num_counters];
```

Comment:
This implementation means that you couldn't instrument lazy imports that happen after fuzzing is started, right?

Is there a way around that? Perhaps by telling libFuzzer there are multiple "modules" and adding a new module when full?

If this can't be resolved, please make errors like this extremely obvious. If a module is imported and attempted to be instrumented after fuzzing is started, the program should exit with an error.

Reply:
> I'm going to confirm that dropping support for Python 2.7 is something we can do. While almost all of Google's code runs on Python 3 now, most of our library code _supports_ both because occasional programs are still written in 2.7. If we can't drop 2.7, we might be able to do something on our end to use "old Atheris" with Python 2.7, or it may be necessary to include _both_ instrumentation techniques in Atheris. I'm hoping we can just drop 2.7 support and call it a day though.

Alright, after discussion, we have no need to support Python 2.7!